tools/check_wav_file_snr.m

```diff
@@ -0,0 +1,355 @@
+function pass = check_wav_file_snr(fn, test_tone_f, logdir)
+
+if nargin < 3
+	logdir = '/tmp';
+end
+
+if nargin < 2
+	% Automatic find of tone frequency
+	test_tone_f = 0;
+end
+
+if nargin < 1
+	error('Need to provide file name to analyze');
+end
+
+if exist('OCTAVE_VERSION', 'builtin')
+	pkg load signal;
+end
+
+%fp = '/home/singalsu/Downloads/';
+%wf = 'bat.wav.3Vbz0k'; % total mess
+%wf = 'bat.wav.zhJhMf'; % glitch in right channel
+%wf = 'bat.wav.Wodg37'; % low quality
+%wf = 'bat.wav.WR5of5'; % Low signal level but OK
+%wf = 'bat.wav.05IgGp'; % good
+%wf = 'bat.wav.3UiPoZ'; % good
+%fn = fullfile(fp, wf);
+
+param.t_ignore_from_start = 30e-3;
```

Comment:
Should be possible to enter the ignorefrom begin and end times from command line. And the test criteria should be printed to see them in log.